### DISCUSSION TITLE
Ask uws.js on hyper-express was support bun ?

### DISCUSSION THREAD

**Discussion by karomahlailatul** [kartikk221/hyper-express#200] [Q&A] [answered]
hello, wanna ask, i am try used bun on based WSL, and try running hyper-express, it was showing ```error: This version of uWS.js supports only Node.js LTS versions 16, 18 and 20 on (glibc) Linux, macOS and Windows, on Tier 1 platforms (https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list).

TypeError: symbol 'napi_register_module_v1' not found in native module. Is this a Node API (napi) module?``` , any can tell me what i miss something ? thank you

**Comment by kartikk221** [comment, marked as answer]
Hello, The library `uWebsockets.js` which `hyper-express` is based on top of does not support Bun at the moment. For this reason, `hyper-express` automatically does not support Bun either. Feel free to follow up or research the `uWebsockets.js` repository to determine future plans and status for Bun compatibility.

When `uWebsockets.js` adds support for Bun, `hyper-express` will also automatically become compatible with Bun.
